cmd/bucket-handlers.go

```diff
@@ -1200,6 +1200,7 @@ func (api objectAPIHandlers) PostPolicyBucketHandler(w http.ResponseWriter, r *h
 		writeErrorResponseHeadersOnly(w, toAPIError(ctx, err))
 		return
 	}
+	opts.WantChecksum = checksum
```

Comment:
@klauspost I believe we only need this single line from the snippet you provided on the issue, since we already AddChecksumNoTrailer here: https://github.com/minio/minio/blob/master/cmd/bucket-handlers.go#L1147-L1152

Correct me if I'm wrong, but the mint test in https://github.com/minio/minio-go/pull/1996 passes with this single line change.